capacitor.config.template.ts

```diff
@@ -35,3 +36,7 @@ const config: CapacitorConfig = {
 };
 
 export default config;
+
+function toBoolean(v: string) {
+  return !!v;
```

Comment:
Nice catch, yeah it would mean manually specifying `"false"` would be interpretted as true. Updated the check with b7d5971646257db14304b0acd8d8cec5f62fa3df